package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-test",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I think we should bump the versino to 0.1.1 since this is not a new feature :)

Reply:
Thanks for the comment!